tests/rpc/testdata/verge.json

```diff
@@ -0,0 +1,54 @@
+{
+    "blockHeight": 3000000,
+    "blockHash": "0000000000016d4250cba4335ffa6bad92d06d677f3dc52ef35d1531e2e58a4c",
+    "blockTime": 1553557428,
+    "blockTxs": [
+        "9c8475a7db618931475c770833e1367bbc602d7228cbdf9c9ff47409d9769376",
+        "37252d35019a161d485eb724a629181ef9315a5743c548a82ad00b72ae33327b"
+    ],
+    "txDetails": {
+        "37252d35019a161d485eb724a629181ef9315a5743c548a82ad00b72ae33327b": {
+            "hex": "010000000001011f9216c16c78386540d7ae7d32657c388ef5f204596f84ea0851dcb78c479a87010000001716001432d094c8e2efd308d2c69affd6712ebbf7a5a286ffffffff0289544a110000000017a91457ca840d6c811dd6808722babe3f88d2fdb2ca14875bb1c9180000000017a91482696a9b4188eda8f93b120315cad4260cbb90db8702483045022100aa256153317133fa719180935017671e33ca77df6f5426554f5b0855f07a392b02202684e8c30623e1c4e753ea23a95bd571e1fda7d15dc0b1e2d54ff5bc50329256012103399a1d98f0733ef400ff8d4f43fe4543065f7f387c863361c77a8826321ca6fb00000000",
+            "txid": "37252d35019a161d485eb724a629181ef9315a5743c548a82ad00b72ae33327b",
+            "blocktime": 1553557428,
+            "time": 1553557346,
+            "locktime": 0,
+            "version": 1,
+            "vin": [
+                {
+                    "txid": "05f10716efd3381e66416c7fd35d53652f1142d63dd381d61aee10e624070dcd",
+                    "vout": 0,
+                    "scriptSig": {
+                        "hex": "473044022029c3bf94ca0c567f57206a1b7671831d1f4c48ff627bc945a0c35296f7c2ae2f0220093d3a266a6598ee5f4e94b984e5c21e1122b81e8448be0d6f0fcdf81923ea6201210341b1117d8402341659664f0629786b4bded37b1a2d2ad43ba0d78dc921cf0a5b"
+                    },
+                    "sequence": 4294967295
+                },
+			{
+                    "txid": "40408287e6de678b04a9d5d1791234cd545fb77b1464a4dc2dc5a8c73e62f2fd",
+                    "vout": 0,
+                    "scriptSig": {
+                        "hex": "004730440220138a8dd272f3081e54b172fb3e6cf43fe5375369d7e93c310ceba852c22ba16f022050de068f932f6d6db9889a8d4e489ffeda2ed1848cd2a71a06d635e6d1799b8e01483045022100af99d412c97062219ef56c3c8ad99023feab94042d310f792392940c61df9bdd02200b4542395624412260c96c46427509c2b801df33e15da08c6c505594c7c142010147522102a39a18546545ee2944481953c25b3a5f4942cd8ed533ae2daabb4339c68c599b210349ed7f7284feff2b9d0b3493fde313afcc63b41347592c435f5408852c9ea05d52ae"
+                    },
+                    "sequence": 4294967295
+                }
+
+            ],
+            "vout": [
+                {
+                    "value": 17230.000000,
+                    "n": 0,
+                    "scriptPubKey": {
+                        "hex": "76a91431ae5739fbb60574e10323f38538ada2730826c388ac"
+                    }get
```

Comment:
get? Is this correct?

Reply:
@wakiyamap no, but doesnt fix the issue im experiencing =\